chapter_convolutional-neural-networks/conv-layer.md

```diff
@@ -6,7 +6,7 @@
 ## 互相关运算
 
 严格来说，卷积层是个错误的叫法，因为它所表达的运算其实是*互相关运算*（cross-correlation），而不是卷积运算。
-根据 :numref:`sec_why-conv`中的描述，在卷积层中，输入张量和核张量通过(**互相关运算**)产生输出张量。
+根据 :numref:`sec_why-conv`中的描述，在卷积层中，输入张量和核张量通过（**互相关运算**）产生输出张量。
```

Comment:
Hi! Thanks for contributing! This special symbol is used to create slides. Please see https://book.d2l.ai/user/slides.html.

Reply:
Got it. Thanks!